app/extensions/contact/gist.py

```diff
@@ -8,5 +8,5 @@ def load():
 class GIST(Extension):
```

Comment:
```suggestion
class Gist(Extension):
```

I noticed you were doing some code quality changes, we could change this to normal capitalization, instead of all caps?

Reply:
good idea, done!